examples/docker-simple-on-ubuntu/terraform.tfvars

```diff
@@ -0,0 +1,11 @@
+resource_group_name = "AzureQuickstart"
+
+resource_group_location = "West US"
+
+admin_username = "azure"
+
+admin_ssh_publickey = "REPLACE_WITH_SSH_PUBLICKEY"
```

Comment:
(same as above, I think it'd make sense to make this the file path and default it to the default public ssh key path?)